### PR TITLE
Update to message recommendations asking to run pro

### PR DESCRIPTION
**Pull request by dheyay**
## Why is this needed?

This PR solves all of our problems because...
Ensure 'sudo' is included for commands running `pro` which require root access.
Fixes: #2598

## Checklist

 - [ ] I have updated or added any unit tests accordingly
 - [ ] I have updated or added any integration tests accordingly
 - [ ] Changes here need to be documented, and this was done in: 

## Does this PR require extra reviews?

 - [ ] Yes
 - [x] No

### CODE REVIEW

**Comment by lucasmoura** [comment]
There are two things missing on this PR:

1) We need to update the existing integration tests that check that message:
https://github.com/canonical/ubuntu-pro-client/blob/main/features/livepatch.feature#L54

2) We need to update the translation files. To do that, just run the `update-pos.sh` script inside the `tools` folder and commit the result. But double check that it has not touched any other message besides the one that was modified.

File: features/livepatch.feature [hunk 1]
```diff
@@ -51,7 +51,7 @@ Feature: Livepatch
         NOTICES
         The current kernel \(5.4.0-(\d+)-kvm, x86_64\) is not supported by livepatch.
         Supported kernels are listed here: https://ubuntu.com/security/livepatch/docs/kernels
-        Either switch to a supported kernel or `pro disable livepatch` to dismiss this warning.
+        Either switch to a supported kernel or `sudo pro disable livepatch` to dismiss this warning.
```
Comment (lucasmoura) [inline features/livepatch.feature:54]: We also need to add this change to line 68

**Review by orndorffgrant — COMMENTED** [review]
Even though it is a logging statement, I think the `pro config set` suggestion in `config.py` around line 588 should have a `sudo` as well.

Otherwise this looks good!

**Comment by dheyay** [comment]
Should be good  to merge now

File: uaclient/config.py [hunk 2]
```diff
@@ -585,7 +585,7 @@ def warn_about_invalid_keys(self):
             LOG.warning('legacy "ua_config" found in uaclient.conf')
             LOG.warning("Please do the following:")
             LOG.warning(
-                "  1. run `pro config set field=value` for each"
+                "  1. run `sudo pro config set field=value` for each"
```
Comment (lucasmoura) [inline uaclient/config.py:588]: It seems we also have that same suggestion on `postinst`. I think we should update it there as well

Comment (dheyay) [reply uaclient/config.py:588]: Should be good now @lucasmoura, thanks!

**Review by orndorffgrant — APPROVED** [review]
Thank you!

**Pull request merged**